types/src/front/lib/assistant.ts

```diff
@@ -50,7 +50,7 @@ export function getSmallWhitelistedModel(
     return GPT_4O_MINI_MODEL_CONFIG;
   }
   if (isProviderWhitelisted(owner, "anthropic")) {
-    return CLAUDE_3_5_SONNET_DEFAULT_MODEL_CONFIG;
+    return CLAUDE_3_5_HAIKU_DEFAULT_MODEL_CONFIG;
```

Comment:
I think this was inaccurate

Reply:
It used to be the old sonnet here, which was then updated to new sonnet.